cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -65,9 +65,13 @@ def export_model_predictions(cloud_event: http.CloudEvent) -> None:
         chunks_ref,
     )
 
-    # Set the error message to the Series' json string for testing
-    # purposes. In the actual implementation, the Series should be stored somewhere.
-    raise NotImplementedError(h3_predictions.to_json())
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(
```

Comment:
FYI we'll need to add the unique id from https://github.com/UrbanSystemsLab/climateiq-frontend/pull/22 here as well